include/hecisupport.h

```diff
@@ -30,8 +30,8 @@
 #ifndef _HECISUPPORT_H_
 #define _HECISUPPORT_H_
 
-#include "../../libkernelflinger/protocol/Heci.h"
-#include "../../libkernelflinger/protocol/MkhiMsgs.h"
+#include "Heci.h"
+#include "MkhiMsgs.h"
 

```

Comment:
in libkernelflinger/rpmb/rpmb_ufs.c and libkernelflinger/rpmb/rpmb_virtual.c
They are using the following style which is "protocal/*.h",  but here protocol is not exist. Suggest to keep the same style which is "protocal/*.h"
 #include "../protocol/ufs.h"
#include "../protocol/ScsiPassThruExt.h"
#include "protocol/ufs.h"
#include "protocol/ScsiPassThruExt.h"